source/_docs/configuration/templating.markdown

```diff
@@ -1091,6 +1091,38 @@ While Jinja natively supports the conversion of an iterable to a `list`, it does
 
 Note that, in Home Assistant, to convert a value to a `list`, a `string`, an `int`, or a `float`, Jinja has built-in functions with names that correspond to each type.
 
+### Iterating multiple objects
+
+The `zip()` function is used to iterate multiple collections in one operation without using an indexer.
```

Comment:
I don't think there is a much simpler way to describe what zip does, but I would simplify it by removing Indexer.

```suggestion
The `zip()` function can be used to iterate over multiple collections in one operation.
```